db/migrate/20250828144553_populate_duplicated_in_advance_fees.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class PopulateDuplicatedInAdvanceFees < ActiveRecord::Migration[8.0]
+  def up
+    query = <<~SQL
+      WITH duplicated_fees AS (
+        SELECT organization_id, pay_in_advance_event_transaction_id, charge_id, charge_filter_id
+        FROM fees
+        WHERE fees.deleted_at IS NULL
```

Comment:
Is the deleted_at check necessary ? Shouldn't it be  present in the query below or in the index ?